lua/scriptwrapper.lua

```diff
@@ -1,5 +1,5 @@
 --[[
-   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
```

Comment:
It is probably better to have one location for the same code. But alternatively you could have also loaded a separate scriptwrappe-no-sandbox.lua.